org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/BUILD.md

```diff
@@ -0,0 +1,46 @@
+# Build
+
+This project uses Maven with Eclipse Tycho. To perform a build open a command line and simple run 
+
+```mvn``` 
+
+By default, this will execute the Maven goals `clean verify`.
+
+## Build Result
+
+Besides all modules being build the main artifact that is produced by the build is a p2 repository. This can be used by Eclipse to install the DSL feature. 
```

Comment:
miss a comma

Reply:
Can be simplified:

The build processes all modules and produces a p2 repository. That repository can be used to install the DSL feature. It is located in `repository/target/repository` and archived to `repository/target`.